Gems/ROS2/Registry/sdfassetbuilder_settings.setreg

```diff
@@ -13,7 +13,12 @@
                     "xacro"
                 ],
                 "UseArticulations": true,
-                "URDFPreserveFixedJoint": true
+                "URDFPreserveFixedJoint": true,
+                "PathPrefixMap": {
+                    "model://": ["./", "../" ],
+                    "package://": ["./", "../" ],
+                    "file://":  ["/"]
```

Comment:
For the `file://` the value of the path should be an empty string.

This is because the `file://` scheme already ascribes a `/` as meaning an [absolute path](https://en.wikipedia.org/wiki/File_URI_scheme)